example/integration_test/app_test.dart

```diff
@@ -87,5 +87,42 @@ void main() {
       expect(tokenText.data?.isNotEmpty, true);
       expect(tokenText.data?.toLowerCase().contains('error'), false);
     });
+
+    testWidgets('fetchInit and fetchExecute runs', (tester) async {
+      app.main();
+      await tester.pumpAndSettle();
+
+      final Finder initFinder = find.byKey(const Key('initClient'));
```

Comment:
This had me confused for a while if it was doing getClient or fetchClient

Reply:
Changed both buttons and keys to getClient and fetchClient to be easier to differentiate